### ISSUE TITLE
UnequipItems

### ISSUE THREAD

**Issue by tvister091** [Demorome/Showoff-NVSE#27]
The noEquip flag does not work in the UnequipItems function

**Comment by Demorome** [comment]
Yep, unfortunately it's an issue I wasn't aware of that exists in the vanilla UnequipItem function.
So you'd have to wait until that vanilla bug gets fixed, and idk how for now.
